examples/scala/src/main/scala/example/Streaming.scala

```diff
@@ -123,23 +128,22 @@ object Streaming {
     // Start reading tbl2 as a stream and do a streaming write to tbl1
     // Prior to Delta 0.5.0 this would throw StreamingQueryException: Detected a data update in the source table. This is currently not supported.
     val stream4 = spark.readStream.format("delta").load(tbl2).writeStream.format("delta")
-      .option("checkpointLocation", new File("/tmp/checkpoint/tbl1").getCanonicalPath)
+      .option("checkpointLocation", new File("/tmp/delta-streaming/checkpoint/tbl1").getCanonicalPath)
       .outputMode("append")
       .start(tbl1)
-
+    
+    Thread.sleep(10 * 1000)
```

Comment:
Not sure how this got added

Reply:
I added this so that the streaming is running properly when we do the repartitioning.